src/soundmanager.h

```diff
@@ -62,13 +62,13 @@ class SoundManager : public QObject {
     // open, this method simply runs through the list of all known soundcards
     // (from PortAudio) and attempts to close them all. Closing a soundcard that
     // isn't open is safe.
-    void closeDevices();
+    void closeDevices(bool wait);
```

Comment:
closeDevices(0) is not more or less readable than closeDevice(false). 
The wait time depends on the device which is closed. So it is here no option to use the time as parameter. 

In general a suitable solution for this is to define a variable: 

bool sleepAfterClosing = false; 
closeDevice(sleepAfterClosing). 

But in the days of hover help on functions, IMHO this is a bit overdone here. 
It is additional code to maintain and the compiler cant check if the name matches still matches the parameter.